interactors/input.js

```diff
@@ -0,0 +1,19 @@
+import { createInteractor, perform, focused } from '@bigtest/interactor';
+import { isVisible } from 'element-is-visible';
+
```

Comment:
Does the `TextField` interactor from `@bigtest/interactor` not work here?

Reply:
Oh sure, that makes sense.